src/lib/expression/evaluation/expression_evaluator.cpp

```diff
@@ -1613,4 +1613,14 @@ std::vector<std::shared_ptr<ExpressionResult<Result>>> ExpressionEvaluator::_pru
   return results;
 }
 
+// We explicitly instantiate these template functions because (at least) clang-12 does not instantiate them for us.
```

Comment:
Maybe also mention the platform macOS here?

Reply:
For this, we should verify that the error does not appear on ubuntu with clang 12.

Reply:
Does the CI use clang 12?

Reply:
> Does the CI use clang 12?

No, it doesn't.

Reply:
Good. Let's leave the comment as is and merge this PR.

Reply:
Any reason not to keep clang9 and use clang12 over clang10 in the CI?

Reply:
> Any reason not to keep clang9 and use clang12 over clang10 in the CI?

Not really but we might have to upgrade the Ubuntu version then. Also, with the Mac ARM CI we will have clang12 as part of the CI.